cypress/integration/pages/frontPage/tests.js

```diff
@@ -134,6 +134,29 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
             },
           );
         });
+        it('should contain Useful Links if usefulLinks block data exists', () => {
+          cy.request(`${config[service].pageTypes.frontPage.path}.json`).then(
+            ({ body }) => {
+              const pageData = body.content.groups;
+              const usefulLinks = pageData.find(data => {
+                return data.type === 'useful-links';
+              });
+              const hasStrapline = usefulLinks
+                ? Object.prototype.hasOwnProperty.call(usefulLinks, 'strapline')
+                : false;
```

Comment:
could be replaced by

```suggestion
	const hasStrapline = usefulLinks && Object.prototype.hasOwnProperty.call(usefulLinks, 'strapline')
```

the `&&` operator basically checks if the check on the left is correct, and if so it will evaluate the code on the right, otherwise returns `false`.